server/Dockerfile

```diff
@@ -19,26 +19,15 @@ RUN apt-get update -qq -y \
 
 WORKDIR ${APP_DIR}
 COPY openapi_server openapi_server/
-RUN mkdir dslim-bert
 COPY requirements.txt prod-requirements.txt uwsgi.ini ./
+COPY save_bert.py ./
 
-#download dslim-bert model
-RUN mkdir dslim-bert/model \
-    && mkdir dslim-bert/tokenizer \
-    && curl -O https://huggingface.co/dslim/bert-base-NER/blob/main/config.json \
-    && curl -O https://huggingface.co/dslim/bert-base-NER/blob/main/pytorch_model.bin \
-    && mv config.json dslim-bert/model \
-    && mv pytorch_model.bin dslim-bert/model \
-    && curl -O https://huggingface.co/dslim/bert-base-NER/blob/main/special_tokens_map.json \
-    && curl -O https://huggingface.co/dslim/bert-base-NER/blob/main/tokenizer_config.json \
-    && curl -O https://huggingface.co/dslim/bert-base-NER/blob/main/vocab.txt \
-    && mv special_tokens_map.json dslim-bert/tokenizer \
-    && mv tokenizer_config.json dslim-bert/tokenizer \
-    && mv vocab.txt dslim-bert/tokenizer \
-#install requirements    
-    && pip install --no-cache-dir \
+#install requirements
+RUN pip install --no-cache-dir \
     -r requirements.txt -r prod-requirements.txt \
-    && useradd --create-home --shell /bin/bash nlp 
+    && useradd --create-home --shell /bin/bash nlp
+# Download BERT model:
+RUN python save_bert.py
```

Comment:
Hadolint may complain about having two consecutive RUN. The solution would be to use one or let hadolint know that it should ignore this issue.